src/Microsoft.DotNet.Build.Tasks/PackageFiles/partialfacades.targets

```diff
@@ -9,35 +9,6 @@
     <IgnoreArchitectureMismatches>true</IgnoreArchitectureMismatches>
   </PropertyGroup>
 
-  <!-- It seems this needs to be set here, and cannot be set in individual project files -->
-  <PropertyGroup Condition="'$(IgnoreArchitectureMismatches)' == 'true'">
-    <!-- Needed because we are using mscorlib.dll from the CoreCLR package, rather than a true reference assembly -->
-    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
-  </PropertyGroup>
-
-  <!-- PostFilterNuGetReferences
-       Filters the set of references generated from the ResolveNugetPackages target. This is a default target for partial facade
-         assemblies, and can be optionally used in other projects as well, by setting PostFilterNugetReferences=true. The target
-         contains some logic for removing well-known assemblies from the "Microsoft.DotNet.CoreCLR" package, except for "mscorlib.dll".
-  -->
-  <Target Name="PostFilterNugetReferences"
```

Comment:
Ah, I see what you are saying. Microsoft.TargetingPack.Private.NETNative has ~20 private assemblies in it. We'll have to think of something to do about that if we don't want all of them exposed in a project. Splitting it into separate NuGet packages is the obvious option, but that's a lot of NuGet packages then.